### PR TITLE
Feature/Implement tappable SwiftUI Text that redirects to link

### PR DESCRIPTION
**Pull request by yllfejziu**

### CODE REVIEW

File: Tests/Tests/Core/Extensions/SwiftUI/TextTests.swift
Comment (markom-povio) [inline Tests/Tests/Core/Extensions/SwiftUI/TextTests.swift:1]: Do we need this test in the `Core` package?
Maybe we can add this component to the Storybook demo project and add UI tests there instead. What do you think?

Comment (yllfejziu) [reply Tests/Tests/Core/Extensions/SwiftUI/TextTests.swift:1]: I agree, let's omit it from `Core` for now.

**Review by markom-povio — APPROVED** [review]
👍

File: Sources/Core/Extensions/SwiftUI/Text+PovioKit.swift [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+//  PovioKit
+//
+//  Created by Yll Fejziu on 27/10/2023.
+//  Copyright © 2023 Povio Inc. All rights reserved.
+//
+
+import SwiftUI
+
+public extension Text {
+  /// Creates an instance of hypelinked Text using markdown formatting syntac
```
Comment (borut-t) [inline Sources/Core/Extensions/SwiftUI/Text+PovioKit.swift:12]: Typo "hypelinked".

**Review by borut-t — APPROVED** [review]
👌

**Pull request merged**